.github/workflows/deploy.yml

```diff
@@ -45,28 +48,64 @@ jobs:
           sushi --version
           sushi .
 
+
+    # Install GOFSH if needed
+      - name: "Install GOFSH"
+        shell: bash
+        run: |
+          npm install -g gofsh@1.6.4
+          gofsh --version
+
+    # Install reproschema2fhir if needed
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
maybe we can stick to checkout@v4? just seems unusual to use both v4 and v2 in the same workflow